build/Common.tests.props

```diff
@@ -5,11 +5,11 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
-  <PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Tests'))">
```

Comment:
Some of the test projects, including E2E, don't end with `.Tests` but with `Tests`.